tests/visual/pager-rtl.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
```

Comment:
I would suggest having both `ltr` and `rtl` pager visual tests in 'pager.html', displayed after a title for the current mode. This way, it will be more convenient to track/see changes in visual tests when needed and reduce png count

Reply:
I have done this for other controls (see #317) and it appeared that I don't like mixing `ltr` & `rtl` in a single html file :). 

I will add this test to the "pager.html" after #329